### ISSUE TITLE
Missing directories

### ISSUE THREAD

**Issue by stefan-jung** [oasis-open/dita-rng-converter#11]
The transformation fails, because `/lib` does not exist. This is related to the following lines in the `build.xml`:

```xml
<property name="toolsDir" location="${basedir}/../../tools"/>
<property name="javaLibDir" location="${basedir}/lib"/>
```

**Comment by nakohdo** [comment]
I can confirm this behavior. The "javaLibDir" property is used only once to create a fileset with Saxon-related Java files:

```XML
<path id="saxonClasspath">
    <fileset dir="${javaLibDir}">
      <include name="**/saxon9*.jar"/>
      <include name="**/xml-resolver*.jar"/>
      <include name="**/resolver*.jar"/>
    </fileset>
</path>
```

**Comment by nakohdo** [comment]
Setting the "javaLibDir" property to point to oXygen's `\lib` directory seems to solve the problem.

```XML
<property name="javaLibDir" location="C:\path to oXygen installation directory\Oxygen XML Editor 19\lib" />
```

**Comment by nakohdo** [comment]
A related problem is the setting of the "saxonDir" property:

`<property name="saxonDir" location="${javaLibDir}/saxon"/>` 
didn't work for me so as a workaround I changed the setting as follows:
`<property name="saxonDir" location="${javaLibDir}" />`

**Comment by drmacro** [comment]
New target OT is 3.x, where the lib structure is different. Will correct code to reflect.

**Comment by drmacro** [comment]
I think I've corrected this by ensuring that the lib/ dir is populated in the git repo and adding the necessary jars.

**Comment by drmacro** [comment]
Closing this as obsolete and/or fixed. Creating a new issue to track update of OT plugins to work with 3.x
